todo.md

```diff
@@ -22,11 +21,10 @@
 ## Popups
 
 * chore: rename popup -> transient
-* fix: default values for options don't get displayed on initial draw
-
-## Stash
-
-* feat: delete stash
+* refactor: action/switch/option/config should all exist on the same list within state, allowing better control over
```

Comment:
Absolutely love this. Having a "list" rather than an implementation makes it much easier to a see and audit

Reply:
Agreed. It's _slightly_ hacky right now with the different columns/rows and headers. Works fine, but I totally agree, a list of components is much simpler and lets you do more.

Currently, something I want to do (and can't) is to have _multiple_ 2D-grids within a popup. The current implementation limits you to the actions group at the bottom, and this works well enough that I would consider it a very low-priority fix, but still.. can't do it.